### PR TITLE
Upgrade to formidable 2.x

### PR DESCRIPTION
**Pull request by AmauryD**
See #200 for the related issue.

### CODE REVIEW

**Review by 3imed-jaberi — APPROVED** [review]
+1

File: index.d.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ declare namespace koaBody {
         /**
          * {String} If you want checksums calculated for incoming files, set this to either 'sha1' or 'md5', default false
          */
-        hash?: string;
```
Comment (divmgl) [inline index.d.ts:43]: What's the purpose of this change? This would break existing `koa-body` implementations in production.

**Review by divmgl — APPROVED** [review]
LGTM, aside from the small property nit. Can we get this in? @dlau

File: index.d.ts (hunk 1 above)
Comment (AmauryD) [reply index.d.ts:43]: The option.hash in formidable has been renamed into option.hashAlgorithm. I can make an adaptation in order to not release a major version.

**Comment by thzero** [comment]
@dlau yes... please get this in.

**Comment by tomoat** [comment]
@MarkHerhold please get this in.

**Comment by MarkHerhold** [comment]
I can't. I'm no longer an owner since the org move.
image

**Comment by jonathanong** [comment]
@diau done!

**Comment by dlau** [comment]
@jonathanong  hmm, it seems we or the @koajs/body-parsing  group also  needs access to this specific repo:
Screen Shot 2022-04-21 at 12 45 38 PM

**Comment by jonathanong** [comment]
oh wow lol. just added

**Comment by dlau** [comment]
It worked, thanks. Merging now, will publish under `5.0` due to the `hash->hashAlgorithm` change in interface.
